### DISCUSSION TITLE
Erro de SSL em rotas do mercado livre

### DISCUSSION THREAD

**Discussion by pauloeasysoft** [viniciussanchez/RESTRequest4Delphi#197] [Q&A]
Estou fazendo um sistema para se comunicar com mercado livre porém quando estou no linux ele fica dando erro de cannot load ssl, porem no windows, funciona, ja colocamos as llibs e tudo mais porem nao funciona de forma alguma, oque estamos suspeitando que possa ter algo no delphi a adicionar ou no propio componente para comunicar, pois o servidor está com tudo liberado.

**Comment by viniciussanchez** [comment]
Desconheço, normalmente é dll isso mesmo
